consensus/src/testing/byzantine.rs

```diff
@@ -108,9 +108,8 @@ impl<'a> MaliciousMember<'a> {
             debug!(target: "malicious-member", "Creating a legit unit for round {}.", round);
             let control_hash = ControlHash::<Hasher64>::new(&parents);
             let new_preunit = PreUnit::<Hasher64>::new(self.node_ix, round, control_hash);
-            let coord = UnitCoord::new(round, self.node_ix);
             if round != self.forking_round {
-                let data = Data::new(coord, 0);
+                let data = 0;
```

Comment:
Just drop it in the constructor (apparently I didn't mark all the occurrences of this problem?).

Reply:
done